lib/index.js

```diff
@@ -326,3 +327,65 @@ exports.createStorageClient = async (ctx, accessToken, storageIdPrefix) => {
 
     return storageClient;
 };
+
+exports.createFusebitFunctionFromExpress = (app, { disableStorageClient } = {}) => {
+    // See https://github.com/fusebit/samples/blob/master/express/index.js#L6
+    Object.setPrototypeOf(Object.getPrototypeOf(Object.getPrototypeOf(app.response)), Mock.Response.prototype);
+    Object.setPrototypeOf(Object.getPrototypeOf(Object.getPrototypeOf(app.request)), Mock.Request.prototype);
+
+    return async (ctx) => {
+        exports.debug('HTTP REQUEST', ctx.method, ctx.url, ctx.headers, ctx.body);
+
+        if (!disableStorageClient) {
+            ctx.storage = await exports.createStorageClient(
+                ctx,
+                ctx.fusebit.functionAccessToken,
+                `boundary/${ctx.boundaryId}/function/${ctx.functionId}/root`
+            );
+        }
+
+        let url = ctx.url.split('/');
+        url.splice(0, 5);
+        url = Url.parse('/' + url.join('/'));
+        url.query = ctx.query;
+        url = Url.format(url);
+
+        let req = new Mock.Request({
+            url,
+            method: ctx.method,
+            headers: ctx.headers,
+        });
+        req.fusebit = ctx;
+        if (ctx.body) {
+            // Simulate the body had already been parsed
+            req._body = true;
+            req.body = ctx.body;
+        }
+
+        return new Promise((resolve, reject) => {
+            try {
+                let responseFinished;
+                let res = new Mock.Response({
+                    onEnd: () => {
+                        if (responseFinished) return;
```

Comment:
Code needs prettier to be executed against it.

Reply:
Done